docusaurus/src/components/HomepageFeatures/index.js

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line no-use-before-define
```

Comment:
Please let's never add ignore to the linter unless is something planned or discussed.